src/app/History.ts

```diff
@@ -1,8 +1,9 @@
 import { createBrowserHistory } from 'history';
+import createMemoryHistory from 'history/createMemoryHistory';
 
 const webRoot = (window as any).WEB_ROOT ? (window as any).WEB_ROOT : undefined;
 const baseName = webRoot && webRoot !== '/' ? webRoot + '/console' : '/console';
-const history = createBrowserHistory({ basename: baseName });
+const history = process.env.TEST_RUNNER ? createMemoryHistory() : createBrowserHistory({ basename: baseName });
```

Comment:
For me, tests are still working without this change... what is it for?

Reply:
Fine, just saw your comment
> `yarn test` should now run beautifully without any warning nor error messages.

Looks good!

Reply:
To add clarity, this is the kind of errors that this is supressing:
![image](https://user-images.githubusercontent.com/23639005/52585078-708e6e80-2df9-11e9-9f82-2f6cf4e3e803.png)

They are ignored but, for me, a little bit annoying